src/context/component-install.js

```diff
@@ -81,58 +82,60 @@ export function subscribe(node, deps, callback) {
  * @param {function(...?)} callback
  */
 export function unsubscribe(node, callback) {
-  removeComponent(node, callback);
+  unmountComponent(node, callback);
 }
 
 /**
- * This hook returns a function that can be used to set a child component. A
- * child component can be set on this component's node or any other in the same
- * tree. When this component is removed, all child components are also removed.
+ * This hook returns a function that can be used to set a managed component. A
+ * managed component can be set on this component's node or any other in the
+ * same tree. When this component is removed, all managed components are also
```

Comment:
what does "same tree" mean? what happens if the component is mounted on a node in a different tree?

Reply:
There can be any number of roots. The obvious cases that we already do: main ampdoc, shadow docs, FIE docs. But also there could be more of those that we don't do now, such as lightbox root.